app.json

```diff
@@ -43,9 +43,6 @@
     },
     "DOC_BASE_URL": {
       "required": true
```

Comment:
in the app.json remove heroku-redis addon

Reply:
it's been removed, don't know how I managed to skip that. thanks